### PR TITLE
Add wheelnav

### PR DESCRIPTION
**Pull request by softwaretailoring**

### CODE REVIEW

File: files/wheelnav/update.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "wheelnav",
+  "repo": "softwaretailoring/wheelnav",
+  "files": {
+    "include": ["wheelnav.js", "wheelnav.min.js", "wheelnav.min.js.map"]
```
Comment (jimaek) [inline files/wheelnav/update.json]: You need to add a `basePath` parameter to `js/dist` for this to work.
Also I dont think wheelnav.js is needed. The min file should be enough

Comment (softwaretailoring) [inline files/wheelnav/update.json]: Thank you for response!
I supposed that the robot reads the 'js/dist' path from package.json.
The update.json example contains the non-minified version, so I believed it is needed.

Comment (softwaretailoring) [inline files/wheelnav/update.json]: I take it already now, I found this page: https://github.com/jsdelivr/libgrabber#add-updatejson-schema
The update.json fixed.

**Comment by jimaek** [comment]
@jsdelivrbot squash and merge

**Pull request closed without merging**